CAE/CorporateEvents/CorporateActions.rdf

```diff
@@ -182,31 +237,44 @@
 		<rdfs:subClassOf rdf:resource="&fibo-cae-ce-act;VoluntaryCorporateAction"/>
 		<rdfs:label xml:lang="en">disclosure action</rdfs:label>
 		<skos:definition xml:lang="en">corporate action involving a requirement for holders or beneficial owners to disclose their name, location and holdings of any issue to the issuer</skos:definition>
```

Comment:
So that notion of voluntary in ISO 20022 has to do with whether or not the shareholder has to do something ... but it is required of the issuer ...  Will need to do some research in another update to address this.

Reply:
changed 'requirement' to request